definitions/subworkflows/pvacseq.cwl

```diff
@@ -106,6 +106,9 @@ inputs:
     vep_to_table_fields:
         type: string[]?
         default: [HGVSc,HGVSp]
+    vep_to_table_prefix:
```

Comment:
Do we actually need this to be an input here since we are not using it in the immuno.cwl? I believe you can just defined this default in the vep.cwl step directly.